### PR TITLE
Fix selection panel positioning

### PR DESCRIPTION
**Pull request by maciaszczykm**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes https://github.com/kubermatic/dashboard/issues/2589

**Special notes for your reviewer**: 

Zrzut ekranu 2020-09-21 o 16 54 46
Zrzut ekranu 2020-09-21 o 16 54 41
Zrzut ekranu 2020-09-21 o 16 54 36
Zrzut ekranu 2020-09-21 o 16 54 54
Zrzut ekranu 2020-09-21 o 16 55 01

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: src/assets/css/material/_main.scss [hunk 1]
```diff
@@ -315,9 +315,9 @@ mat-form-field.mat-form-field-should-float:not(.mat-focused):not(.ng-invalid):no
 
 .mat-select-panel {
   margin-left: 6px;
-  margin-top: 30px;
-  max-width: calc(100% + 20px);
-  min-width: calc(100% + 20px);
+  margin-top: 38px;
+  max-width: calc(100% + 20px) !important;
```
Comment (floreks) [inline src/assets/css/material/_main.scss:319]: Is max-width applied programmatically on this element to use important?

Comment (maciaszczykm) [reply src/assets/css/material/_main.scss:319]: Yes, through `style` attribute.

**Comment by maciaszczykm** [comment]
Fixed comboboxes:

Zrzut ekranu 2020-09-22 o 10 01 11
Zrzut ekranu 2020-09-22 o 09 58 32

**Pull request merged**